parameter/FloatingPointParameterType.cpp

```diff
@@ -264,8 +265,8 @@ bool CFloatingPointParameterType::fromBlackboard(
         CParameterAccessContext& /*ctx*/) const
 {
     // Move from "raw memory" value space to real space
-    // Using an intermediary reference variable to avoid klocwork false positive
-    const float &fValue = reinterpret_cast<const float &>(uiValue);
+    float fValue;
+    utility::binaryCopy(uiValue, fValue);
```

Comment:
Lets pretend that binary returns the copy:
~~~cpp
auto fValue = utility::binaryCopy<float>(dUserValue);
~~~
I prefer to initialize variable on declaration.